taxi/views.py

```diff
@@ -79,6 +106,24 @@ class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
     success_url = reverse_lazy("taxi:car-list")
 
 
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    fields = [
+        "username",
+        "password",
```

Comment:
Including 'password' in the fields list for `DriverCreateView` might not handle password hashing securely. Consider using Django's built-in user creation forms or methods that handle password hashing automatically.